src/installation/guides/zfs.md

```diff
@@ -0,0 +1,159 @@
+provides comparable instructions.
+
+## Traditional bootloaders
+
+For those that wish to forego ZFSBootMenu, it is possible to bootstrap a Void
+system with another bootloader. To avoid unnecessary complexity, systems that
+use bootloaders other than ZFSBootMenu should plan to use a separate `/boot`
+that is located on an ext4 or xfs filesystem. Installing Void to a ZFS root
+requires an installation medium with ZFS drivers. It is possible to build a
+custom image from the official
+[void-mklive](https://github.com/void-linux/void-mklive) repository by running
+the `mklive.sh` script with the command-line argument `-p zfs`. However, for
+`x86_64` systems, it may be more convenient to fetch a pre-built
+[hrmpf](https://github.com/leahneukirchen/hrmpf) image. These images are
+extensions of the standard Void live images and include pre-compiled ZFS modules
+in addition to other useful tools.
+
+After booting a live image with ZFS support, [partition your
+disks](../live-images/partitions.md). The considerations in the partitioning
+guide apply to ZFS installations as well, except that
+
+- The boot partition should be considered necessary unless you intend to use
+   `gummiboot`, which expects that your EFI system partition will be mounted at
+   `/boot`. (This alternative configuration will not be discussed here.)
+- Aside from any EFI system partition, GRUB BIOS boot partition, swap or boot
+   partitions, the remainder of the disk should typically be a single partition
+   with type code `BF00` that will be dedicated to a single ZFS pool. There is
+   no benefit to creating separate ZFS pools on a single disk.
+
+As needed, format the EFI system partition using
+[mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8) and the the boot partition
+using [mke2fs(8)](https://man.voidlinux.org/mke2fs.8) or
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8). Initialize any swap space
+using [mkswap(8)](https://man.voidlinux.org).
+
+### Create a ZFS pool
+
+Create a ZFS pool on the partition created for it using
+[zpool(8)](https://man.voidlinux.org/zpool.8). For example, to create a pool on
+`/dev/disk/by-id/wwn-0x5000c500deadbeef-part3`:
+
+```
+# zpool create -f -o ashift=12 \
+    -O compression=lz4 \
+    -O acltype=posixacl \
+    -O xattr=sa \
+    -O relatime=on \
+    -o autotrim=on \
+    -m none zroot /dev/disk/by-id/wwn-0x5000c500deadbeef-part3
+```
+
+Adjust the pool (`-o`) and filesystem (`-O`) options as desired, and replace the
+partition identifier `wwn-0x5000c500deadbeef-part3` with that of the actual
+partition to be used.
+
+**NOTE:** When adding disks or partitions to ZFS pools, it is generally
+advisable to refer to them by the symbolic links created in `/dev/disk/by-id` or
+(on UEFI systems) `/dev/disk/by-partuuid` so that ZFS will identify the right
+partitions even if disk naming should change at some point. Using traditional
+device nodes like `/dev/sda3` may cause intermittent import failures.
```

Comment:
What would you think about making this a quote similar to the [libvirt page](https://docs.voidlinux.org/config/containers-and-vms/libvirt.html)

Reply:
Done